### DISCUSSION TITLE
When is support Safari Browser in Iphone?

### DISCUSSION THREAD

**Discussion by Jooolee** [foxglove/community#763] [Q&A]
In IPhone is not support:
![image](https://github.com/foxglove/community/assets/37429047/ce27ec51-9fdf-4ccb-a170-63bed1ffb220)

In Honor Magic4 is ok:
![image](https://github.com/foxglove/community/assets/37429047/98496f36-ce68-4b8d-81a3-7e17490131ba)

**Comment by defunctzombie** [comment]
We only support Chrome at this time.

**Reply by jtbandes** [reply]
See this discussion which is linked from the compatibility warning: https://github.com/orgs/foxglove/discussions/174
